src/main/resources/application.yml

```
@@ -16,6 +16,26 @@ spring:
           maxConnectionLifeTimeMS: \${MONGODB_CONNECTIONPOOL_MAX_CONNECTION_LIFE_MS:0}
           maxConnectionIdleTimeMS: \${MONGODB_CONNECTIONPOOL_MAX_CONNECTION_IDLE_MS:120000}
           maxConnecting: \${MONGODB_CONNECTIONPOOL_MAX_CONNECTING:2}
+  cloud:
+    function:
+      definition: notification
```

the function should have the same name of the binding
```suggestion
      definition: notificationQueue
```